### PR TITLE
Data analysis topics

### PR DESCRIPTION
**Pull request by ganicke**
This is a group of pages to replace the _data-logging.md_ page. Let me know what features need mention. I'm calling the chart and console view as the "Data Viewer". What are we calling it for reals?

### CODE REVIEW

File: docs/device/data-analysis/generating.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# Generating data
+
+## Sensor values
```
Comment (pelikhan) [inline docs/device/data-analysis/generating.md]: Thé simplest to generate data is ´´led.plotBarGraph’´.

Comment (ganicke) [reply docs/device/data-analysis/generating.md]: I got it at the end of the 'writing data' topic. I'll move it up to the top if you think it should go there.

**Comment by ganicke** [comment]
Oh, yea. I can do something for remote data collection.

File: docs/device/data-analysis/generating.md (hunk 1 above)
Comment (pelikhan) [reply docs/device/data-analysis/generating.md]: It should be their go to block for Data logging.

Comment (ganicke) [reply docs/device/data-analysis/generating.md]: Ok, I'll re-cast it as the first method to use.

**Pull request merged**